Samples/outlook-check-item-categories/README.md

```diff
@@ -18,7 +18,11 @@ description: "Use Outlook Smart Alerts to verify that required color categories
 
 # Use Outlook Smart Alerts
 
-**Applies to**: Outlook on Windows
+**Applies to**: Outlook on Windows | Outlook on Mac (new UI)
+
+> **Note**: Although the Smart Alerts feature is supported in Outlook on the web, Windows, and new Mac UI (see the "Supported clients and platforms" section of [Use Smart Alerts and the onMessageSend and OnAppointmentSend events in your Outlook add-in](https://learn.microsoft.com/office/dev/add-ins/outlook/smart-alerts-onmessagesend-walkthrough#supported-clients-and-platforms)), this sample only runs in Outlook on Windows and Mac.
```

Comment:
Should move this note (or info) into the applies to section. If at top like this it gets surfaced in search engines as preview for someone trying to read what the sample is about. the summary should be first for SEO.